src/frontend/sharedComponents/BGMapCard.tsx

```diff
@@ -20,57 +18,42 @@ const m = defineMessages({
     defaultMessage: "MB",
     description: "The abbreviation for megabyte",
   },
+  unamedStyle: {
+    id: "sharedComponents.BGMapCard.unamedStyle",
+    defaultMessage: "Unamed Style",
```

Comment:
```suggestion
  unnamedStyle: {
    id: "sharedComponents.BGMapCard.unnamedStyle",
    defaultMessage: "Unnamed Style",
```

Reply:
Im going to keep the ID the same because some of this has been translated in crowdin already